app/views/cards/_form.html.erb

```diff
@@ -1,4 +1,11 @@
-<%= simple_form_for [current_project, @board, @list, @card] do |f| %>
+<%= simple_form_for [current_project, @board, @list, @card], html: {
+  data: {
+    'as-project-id': current_project.id,
```

Comment:
can't you nest this?

```
data: {
  as: {
    project_id: current_project.id,
    resource_id:...
  }
}
```

Reply:
Yeah I thought so too. It did not work. It mangled a stringified `Hash#inspect` into the `as` data attribute.